ha/core/system_health/system_health_const.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
We can keep the name same - const.py. It's already under system_health package.

Reply:
Year should 2021.

Reply:
Done